src/app/payments/update-pending-payments.ts

```diff
@@ -134,9 +134,9 @@ const updatePendingPayment = wrapAsyncToRunInSpan({
 
     // If we had PaymentLedgerType => no need for checking the fields
     if (!paymentHash)
-      throw new InconsistentDataError("paymentHash missing from payment transaction")
+      return new InconsistentDataError("paymentHash missing from payment transaction")
```

Comment:
I'm not sure about this - isn't this really an illegal state @dolcalmi ?

Reply:
uhmm this requires additional changes to register the error correctly. I will do it in another PR